src/jit-diff/jit-diff.cs

```diff
@@ -178,7 +179,7 @@ public Config(string[] args)
                     syntax.DefineOption("t|tag", ref _tag, "Name of root in output directory. Allows for many sets of output.");
                     syntax.DefineOption("c|corelib", ref _corelib, "Diff System.Private.CoreLib.dll.");
                     syntax.DefineOption("f|frameworks", ref _frameworks, "Diff frameworks.");
-                    syntax.DefineOption("m|metric", ref _metric, false, "Metric to use for diff computations (default: CodeSize).");
+                    syntax.DefineOption("m|metric", ref _metric, false, "Metric to use for diff computations. Available metrics: CodeSize(default), PerfScore, PrologSize");
```

Comment:
Can you specify multiple metrics? If so, how?

Reply:
No Andy changes only support a single metic

Reply:
And it is case sensitive

Reply:
It would not be too hard. Jit-diff could invoke jit-analyze once for each metric, or we could update jit-analyze to allow multiple metrics too.